src/chat/ChatScreen.js

```diff
@@ -196,7 +197,24 @@ export default function ChatScreen(props: Props): Node {
   );
 
   return (
-    <KeyboardAvoider style={[componentStyles.screen, { backgroundColor }]} behavior="padding">
+    <KeyboardAvoider
+      style={[componentStyles.screen, { backgroundColor }]}
+      behavior="padding"
+      keyboardVerticalOffset={
+        // This is a slight misuse of this prop: this value is not (to quote
+        // from the jsdoc) "How much the top of `KeyboardAvoider`'s layout
+        // *parent* is displaced downward from the top of the screen." That
+        // value would be zero.
+        //
+        // This addresses an issue where the strip that covers the bottom
+        // inset would scoot up above the keyboard with the rest of the
+        // compose box when the keyboard opens, leaving a blank strip of
+        // wasted space ( #3370). We haven't yet found a better way to avoid
+        // this, while still letting `ComposeBox` control how it occupies
+        // the inset (like how our header components occupy the top inset).
+        -useSafeAreaInsets().bottom
+      }
```

Comment:
Hmm fascinating.

What does this look like when the keyboard *isn't* open? Does it cause the compose box to be lowered into the inset area, where it's obscured at the bottom? On its face it seems like it would.

It looks like this would become our one use of this `keyboardVerticalOffset` prop. If something like this is right, then perhaps the jsdoc on it isn't right and should be edited? It looks like we added that prop in 70eca0716ed0bfc4d3c5c7ac4b4af3a9a80ce121 / #4683 and have never yet used it in main. (It subsequently appeared in #4893, in a previous revision of this same commit.)